IPython/core/ultratb.py

```diff
@@ -731,7 +735,7 @@ def format_record(self, frame_info):
         if lvals_list:
             lvals = '%s%s' % (indent, em_normal.join(lvals_list))
 
-        result = "%s, %s\n" % (link, call)
+        result = "%s%s%s\n" % (link, ", " if call else "", call)
```

Comment:
Could you please redo this to an f-string too? Also the ones at L493, L485, L470?

Reply:
I simplified the logic to avoid repetition for the final case, and converted all in that function to f-strings.